kod/object/active/holder/nomoveon/battler/monster/iceper.kod

```diff
@@ -64,19 +100,40 @@ properties:
 
    piAnimation = ANIM_NONE
    piDrawFX = DRAWFX_TRANSLUCENT_50
-   
+
+   prWeaponOverlay = iceperson_sword % sword as default weapon
```

Comment:
Add at least one more space between the code and comment to make it easier to read.